<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALmoviesRenamer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/brand.png</normaloff>icons/brand.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="panel_loading" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_loading">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_loading">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="table_movies">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Original name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New name</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stack_movie">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Movie title:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_movie_titles">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Not the right movie?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_manual_title_search">
            <property name="text">
             <string>Search it manually...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/magnifier.png</normaloff>icons/magnifier.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="stack_title_search">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="text_title_search">
              <property name="font">
               <font>
                <family>DejaVu Sans Mono</family>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_title_search">
              <property name="text">
               <string>Search...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/magnifier.png</normaloff>icons/magnifier.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_3">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Searching for movie information...</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progress_searching">
              <property name="maximum">
               <number>0</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_5">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>I cannot find that movie.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_title_new_research">
              <property name="text">
               <string>Search again...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/magnifier.png</normaloff>icons/magnifier.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Year:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Director:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Run Time:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_year">
            <property name="text">
             <string>2000</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_director">
            <property name="text">
             <string>a. director</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="combo_runtime"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Canonical title:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_canonical_title">
            <property name="text">
             <string>title, the</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>AKA:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="combo_aka"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Language:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="combo_language">
            <item>
             <property name="text">
              <string>EN</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ES</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IT</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#005000;&quot;&gt;This movie has been correctly renamed, and can be removed from list.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>There has been the following error during renaming:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_error">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>507</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuMovies">
    <property name="title">
     <string>Movies</string>
    </property>
    <addaction name="action_add_movies"/>
    <addaction name="action_add_all_movies_in_folder"/>
    <addaction name="action_add_all_movies_in_folder_subfolders"/>
    <addaction name="separator"/>
    <addaction name="action_remove_selected_movies"/>
    <addaction name="action_remove_all_movies"/>
    <addaction name="separator"/>
    <addaction name="action_change_rename_pattern"/>
    <addaction name="separator"/>
    <addaction name="action_rename_movies"/>
   </widget>
   <addaction name="menuMovies"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_add_movies"/>
   <addaction name="action_remove_selected_movies"/>
   <addaction name="separator"/>
   <addaction name="action_change_rename_pattern"/>
   <addaction name="separator"/>
   <addaction name="action_rename_movies"/>
  </widget>
  <action name="action_about">
   <property name="icon">
    <iconset>
     <normaloff>icons/information.png</normaloff>icons/information.png</iconset>
   </property>
   <property name="text">
    <string>About ALmoviesRenamer</string>
   </property>
   <property name="toolTip">
    <string>About ALmoviesRenamer</string>
   </property>
  </action>
  <action name="action_add_movies">
   <property name="icon">
    <iconset>
     <normaloff>icons/movie_add.png</normaloff>icons/movie_add.png</iconset>
   </property>
   <property name="text">
    <string>Add movies...</string>
   </property>
  </action>
  <action name="action_add_all_movies_in_folder">
   <property name="icon">
    <iconset>
     <normaloff>icons/movies_from_folder.png</normaloff>icons/movies_from_folder.png</iconset>
   </property>
   <property name="text">
    <string>Add all movies in folder...</string>
   </property>
  </action>
  <action name="action_add_all_movies_in_folder_subfolders">
   <property name="icon">
    <iconset>
     <normaloff>icons/movies_from_folder.png</normaloff>icons/movies_from_folder.png</iconset>
   </property>
   <property name="text">
    <string>Add all movies in folder (including subfolders)...</string>
   </property>
  </action>
  <action name="action_remove_selected_movies">
   <property name="icon">
    <iconset>
     <normaloff>icons/movie_remove.png</normaloff>icons/movie_remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove selected movies from list</string>
   </property>
   <property name="toolTip">
    <string>Remove selected movies from list</string>
   </property>
  </action>
  <action name="action_remove_all_movies">
   <property name="icon">
    <iconset>
     <normaloff>icons/movie_erase.png</normaloff>icons/movie_erase.png</iconset>
   </property>
   <property name="text">
    <string>Remove all movies from list</string>
   </property>
   <property name="toolTip">
    <string>Remove all movies from list</string>
   </property>
  </action>
  <action name="action_change_rename_pattern">
   <property name="icon">
    <iconset>
     <normaloff>icons/tag.png</normaloff>icons/tag.png</iconset>
   </property>
   <property name="text">
    <string>Change renaming rule</string>
   </property>
   <property name="toolTip">
    <string>Change  renaming rule</string>
   </property>
  </action>
  <action name="action_rename_movies">
   <property name="icon">
    <iconset>
     <normaloff>icons/pencil.png</normaloff>icons/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Rename movies</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
